tests/document.test.js

```diff
@@ -98,16 +114,127 @@ describe('## Document APIs', () => {
     });
   });
 
+  describe('# ContentType filter /api/documents?contentType=...', () => {
+    it('should get document list according to the filter', (done) => {
+      request(app)
+        .post('/api/documents')
+        .send(document1)
+        .then((res) => {
+            document1 = res.body;
+        })
+        .catch(done);
+
+      request(app)
+        .get(`/api/documents`)
+        .query({ contentType: contentType2 })
+        .expect(httpStatus.OK)
+        .then((res) => {
+          expect(res.body).to.be.an('array');
+          for(let i = 0; i<res.body.length; i++) {
+            expect(res.body[i].contentType).to.equal(contentType2);
+          }
+          done();
+        })
+        .catch(done);
+    });
+  });
+
+  describe('# Created filter /api/documents?creationStart=...&creationEnd=...', () => {
```

Comment:
En fait, pour tous les tests de filtrage, tu peux avoir une seule et unique `describe`, comme `describe('# Filtering documents by query string', () => {` et mettre tous tes `it()` dedans.